From 7aa27fa76f75ecbbbe4988265b989682db18e4af Mon Sep 17 00:00:00 2001
From: Martin Weinelt <hexa@darmstadt.ccc.de>
Date: Fri, 2 Nov 2018 20:52:01 +0100
Subject: [PATCH 11/13] ramips: fix leds on GL.iNet GL-MT300N-V2

The WAN LED now shows the link state. It's color is green,
not blue.

Signed-off-by: Martin Weinelt <hexa@darmstadt.ccc.de>
---
 target/linux/ramips/base-files/etc/board.d/01_leds | 1 +
 target/linux/ramips/dts/GL-MT300N-V2.dts           | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/target/linux/ramips/base-files/etc/board.d/01_leds b/target/linux/ramips/base-files/etc/board.d/01_leds
index 54504c6ee8..6057275978 100755
--- a/target/linux/ramips/base-files/etc/board.d/01_leds
+++ b/target/linux/ramips/base-files/etc/board.d/01_leds
@@ -210,6 +210,7 @@ gl-mt750)
 	;;
 gl-mt300n-v2)
 	set_wifi_led "$boardname:red:wlan"
+	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x1"
 	;;
 hc5661|\
 hc5661a)
diff --git a/target/linux/ramips/dts/GL-MT300N-V2.dts b/target/linux/ramips/dts/GL-MT300N-V2.dts
index e99d5578f0..98c9a79633 100644
--- a/target/linux/ramips/dts/GL-MT300N-V2.dts
+++ b/target/linux/ramips/dts/GL-MT300N-V2.dts
@@ -28,7 +28,7 @@
 		};
 
 		wan {
-			label = "gl-mt300n-v2:blue:wan";
+			label = "gl-mt300n-v2:green:wan";
 			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
 		};
 
-- 
2.11.0

